src/axom/sidre/core/View.hpp

```diff
@@ -383,6 +384,12 @@ class View
    */
   bool isEquivalentTo(const View* other) const;
 
+  /*!
+   * \brief Returns true if both Views are either associated with a buffer or
+   * external, they span the same number of bytes and have unit stride.
```

Comment:
Should the data type matter for update-ability ?  E.g. moving floats to ints?

Reply:
I have a warning in `View::updateFrom` but I'm not sure if we want to prevent it outright.